sources/academy/webscraping/scraping_basics_python/04_downloading_html.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Downloading HTML with Python
+sidebar_label: Downloading HTML
+description: Lesson about building a Python application for watching prices. Using the HTTPX library to download HTML code of a product listing page.
+sidebar_position: 4
+slug: /scraping-basics-python/downloading-html
+---
+
+**In this lesson we'll start building a Python application for watching prices. As a first step, we'll use the HTTPX library to download HTML code of a product listing page.**
+
+---
+
+Using browser tools for developers is crucial for understanding the structure of a particular page, but it's a manual task. Let's start building our first automation, a Python program which downloads HTML code of the product listing.
+
+## Starting a Python project
+
+Before we start coding, we need to setup a Python project. Create new directory with a virtual environment, then inside the directory and with the environment activated, install the HTTPX library:
```

Comment:
small nit, shouldn't this be `set up` not `setup`?

Reply:
You are right! I never thought about it. The dictionary says that:

- set up is a _verb_
- setup or set-up is a _noun_

Reply:
Addressed in eacc4e1f